app-forensics/bulk_extractor/files/fix_call_of_overloaded_errors.patch

```diff
@@ -0,0 +1,24 @@
+diff -ur a/src/scan_exiv2.cpp b/src/scan_exiv2.cpp
+--- a/src/scan_exiv2.cpp	2014-09-16 22:34:00.000000000 +0400
++++ b/src/scan_exiv2.cpp	2019-03-17 08:38:29.479753464 +0300
+@@ -68,7 +68,7 @@
+  * Used for helping to convert libexiv2's GPS format to decimal lat/long
+  */
+ 
+-static double stod(string s)
++static double sub_stod(string s)
```

Comment:
my `c++` knowledge is not enough here, but if you rename this function, the compiler will not report errors.
^_^